keyboards/redragon/k667/info.json

```diff
@@ -0,0 +1,240 @@
+{
+    "keyboard_name": "k667",
+    "manufacturer": "redragon",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "rgb_matrix": true,
+        "console": true,
+        "encoder": true,
+        "nkro": true
+    },
+    "bootmagic": {
+        "matrix": [1, 0]
+    },
+    "usb": {
+        "vid": "0x03F6",
+        "pid": "0x0020",
+        "device_version": "0.0.1"
+    },    
+    "dynamic_keymap": {
+        "layer_count": 2
+    },
```

Comment:
Is there a reason as to why keymap layers are limited to 2?

Reply:
Because this keyboard is designed to support only Windows, it is set up with only two layers.Thanx!